e-commerce/client/src/components/Cart.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import CartList from '../components/Cartlist'
+
+import { Product } from '../App'
+
+type CartProp = {
+    cartLists: Product[];
+    handleAddToCart: (clickedProduct: Product) => void
+    handleRemoveFromCart: (id: number) => void
+}
+
+const Cart:React.FunctionComponent<CartProp> = ({ cartLists, handleAddToCart, handleRemoveFromCart }) => {
```

Comment:
So instead of calling your props `handleAddToCart` or `handleRemoveFromCart`, its best to call them `onAddToCart`, `onRemoveFromCart`. Much better because all event listeners in react start with `on` and not `handle`.

Reply:
Done.